pkg/controller/certificaterequest/update_status.go

```diff
@@ -42,8 +42,7 @@ func (r *ReconcileCertificateRequest) updateStatus(reqLogger logr.Logger, cr *ce
 			cr.Status.IssuerName != certificate.Issuer.CommonName ||
 			cr.Status.NotBefore != certificate.NotBefore.String() ||
 			cr.Status.NotAfter != certificate.NotAfter.String() ||
-			cr.Status.SerialNumber != certificate.SerialNumber.String() ||
-			cr.Status.Status != "Success" {
```

Comment:
This seems unrelated to the rest of the PR. Was this intended?

Reply:
Not related but I accidentally committed bug fix to my feature branch. This removes always catching a non "Status: Success" and setting it to "Success".